tests/pr_test.go

```diff
@@ -2,43 +2,88 @@
 package test
 
 import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
 	"testing"
 
+	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/stretchr/testify/assert"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"gopkg.in/yaml.v3"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
-const defaultExampleTerraformDir = "examples/default"
+// Resource groups are maintained https://github.ibm.com/GoldenEye/ge-dev-account-management
```

Comment:
These references to internal goldeneye might not be required. Same as below in line 18